documents/charter.md

```diff
@@ -1,23 +1,24 @@
-Video traffic is already 70% of the overall traffic volume on the Internet and is expected to grow to 80% by 2028.
+Video traffic is 70% of the overall traffic volume on the Internet and is expected to grow to 80% by 2028.
 Across developed and emerging markets video traffic forms 50-80% of traffic volume on mobile networks.
 New formats like short form videos have seen tremendous growth in recent years.
 These growth trends are likely to increase with new populations coming online on mobile-first markets.
 
-Mobile network operators continuously invest in network resources, including deployment of new generations or new bands of spectrum.
-Since spectrum is a limited and expensive resource, operators often make use of flow-based traffic handling such as shaping of video traffic, especially when the network is highly loaded.
-Operators cannot explicitly measure the degradation that shaping causes to end user quality of experience (QoE), making this approach open loop. 
+While mobile network operators continuously invest in new network capacity, including deployment of new generations or new bands of spectrum, these efforts are not keeping pace with growth in demand. These network operators have found it faster and less expensive to invest in shaping of video traffic on a per-flow basis, which negatively affects video stream quality. Because that traffic is encrypted, network operators cannot explicitly measure the degradation that shaping causes to end user quality of experience (QoE), making this approach open loop. 
```

Comment:
Cellular radio throughput depends on signal strength. Capacity is a factor in this, but other factors include device position within cell (e.g. cell edge, and resulting signal power attenuation per inverse square law), allocated band (e.g. a user indoors being allocated a low throughput band), local interference, and cell handover.  My suggestion is to prepend the following to the start of the paragraph:

"Local mobile radio conditions may constrain the maximum throughput for a given client, or be so volatile as to rapidly change the maximum throughput throughout the course of a session."

Reply:
If a lot of devices are routinely at the cell edge, more cells can be deployed (capacity). 

In any case, I think your proposed prepended text is fine. Maybe you end up with something like this (assuming I understand where you wanted the text)?

> Local mobile radio conditions may constrain the maximum throughput for a given client, or be so volatile as to rapidly change the maximum throughput throughout the course of a session. In addition, despite capacity augmentation work such as deployment of new generations or new bands of spectrum, capacity augmentation efforts are not keeping pace with growth in demand. These network operators have found it faster and less expensive to invest in shaping of video traffic on a per-flow basis, which negatively affects video stream quality. Because that traffic is encrypted, network operators cannot explicitly measure the degradation that shaping causes to end user quality of experience (QoE), making this approach open loop.

Reply:
Thanks, yes that looks good.

Reply:
I'm putting this text in a PR for Issue #72.